### PR TITLE
Some cctld to be linkified

### PR DESCRIPTION
**Pull request by jugyo**

### CODE REVIEW

File: src/com/twitter/Regex.java [hunk 1]
```diff
@@ -94,15 +94,16 @@
       "台灣|新加坡|香港|한국" +
       ")(?=[^\\p{Alnum}@]|$))";
   private static final String URL_PUNYCODE = "(?:xn--[0-9a-z]+)";
+  private static final String SPECIAL_URL_VALID_CCTLD = "(?:(?:" + "co|tv" + ")(?=[^\\p{Alnum}@]|$))";
```
Comment (KL-7) [inline src/com/twitter/Regex.java:97]: Can we avoid string concatenation here and just use the final `"(?:(?:co|tv)(?=[^\\p{Alnum}@]|$))"` string?

Comment (jugyo) [inline src/com/twitter/Regex.java:97]: Yes, we can. I was a little worried when we will have many ccTLD in here in the future :)

Comment (KL-7) [inline src/com/twitter/Regex.java:97]: Oh, I see. You can put them in a separate constant right away then. It's up to you though.

**Comment by KL-7** [comment]
:shipit:

File: src/com/twitter/Regex.java (hunk 1 above)
Comment (bigloser) [inline src/com/twitter/Regex.java:97]: I like the fact that it is split, easier to read :)

Comment (jugyo) [inline src/com/twitter/Regex.java:97]: Indeed, having many same string like `")(?=[^\\p{Alnum}@]|$))"` isn't cool :P

**Comment by jakl** [comment]
Make sure there are at least tickets in rb/js/objc/conformance
:shipit:

**Comment by jugyo** [comment]
Yes, rb/js/objc are remaining.

**Pull request merged**